governance/program/src/state/enums.rs

```diff
@@ -152,11 +152,19 @@ pub enum VoteThreshold {
     /// In other words a '+1 vote' tie breaker is always required to have a successful vote
     YesVotePercentage(u8),
 
-    /// The minimum number of votes in % out of the entire pool of governance tokens eligible to vote
-    /// which must be cast for the vote to be valid
-    /// Once the quorum is achieved a simple majority (50%+1) of Yes votes is required for the vote to succeed
-    /// Note: Quorum is not implemented in the current version
-    QuorumPercentage(u8),
+    /// The minimum number of votes in basis points (1/10000) out of the entire pool of governance tokens eligible to vote
+    /// which must be cast for the proposal to be considered valid.
+    /// Once the quorum is achieved then pass level (in %) has to be casted for an option to succeed,
+    /// if defined to 50% it's a simple majority (50%+1) of Yes votes for the option to succeed
+    AttendanceQuorum {
```

Comment:
better to only add a new enum type, and mark the old one as deprecated this way it's easier to reason about backwards compatibility

Reply:
also: why allow dead_code? the code is used no?